.github/workflows/e2e-parallel-destroy.yml

```diff
@@ -113,30 +111,12 @@ jobs:
       - name: Terraform Job
         uses: hashicorp/setup-terraform@v1
         with:
-          terraform_version: 1.0.11
-
-      - name: Terraform Fmt
```

Comment:
Because this is the destroy workflow, we shouldn't need to format, validate, plan - just tear it all down